docker/dev/xdebug.ini

```diff
@@ -0,0 +1,7 @@
+[xdebug]
+zend_extension=xdebug.so
+xdebug.mode=debug
+xdebug.start_with_request=yes
```

Comment:
Never do this. Allow the [triggers](https://xdebug.org/docs/all_settings#trigger_value) to function as they should.